README.md

```diff
@@ -114,9 +111,16 @@ Once running, the IQ Server product license must be installed. This should be do
 
 Default admin credentials are: `admin` / `admin123`
 
-The IQ Server product license is stored using Java preferences API. By default, the directory location is already customized by a Java system property to be under the sonatype-work directory so as to survive image restarts.
+### Version 96 or Later
+The IQ Server product license is stored in the database and so will survive image restarts as long as the database is
+persisted.
+
+### Version 95 or Earlier
+The IQ Server product license is stored using Java preferences API. By default, the directory location is already
+customized by a Java system property to be under the sonatype-work directory i.e.
+`-Djava.util.prefs.userRoot=${SONATYPE_WORK}/javaprefs` so as to survive image restarts.
 
-If customized using `JAVA_OPTS`, The absolute path to user prefs must point to an already created directory readable by the user account owning the repository manager process. 
+If customized using `JAVA_OPTS`, The absolute path to user prefs must point to an already created directory readable by the user account owning the process. 
```

Comment:
"..., **t**he absolute ..."

Reply:
> "..., **t**he absolute ..."

Fixed in https://github.com/sonatype/docker-nexus-iq-server/pull/46/commits/62b73d3a5891d70bf5f91accb04921038597e481